src/mixins/filing-template-mixin.ts

```diff
@@ -1137,9 +1137,10 @@ export default class FilingTemplateMixin extends DateMixin {
   private buildFolioNumber (filing: AlterationFilingIF | ChgRegistrationFilingIF | SpecialResolutionFilingIF): any {
     const fn = this.getFolioNumber
     const tfn = this.getTransactionalFolioNumber
+    filing.header.folioNumber = fn
```

Comment:
Why is this needed? I think it's already set in the "build" functions above.

Reply:
IMO since this functions responsibility is building the folio Number it should happen here

Reply:
Well now it's in two places.

Reply:
I see that folio number is set from different places like staff payment, in the beginning of building a filing etc. If this method is supposed to set only transactional folio number it would be intuitive if the method name is like buildTransactionalFolioNumber or  setTransactionalFolioNumber.

Reply:
Yes, I agree. You can change it if you want.